openid-federation-1_0.xml

```diff
@@ -6057,17 +5982,28 @@ GET /authorize?
             <t>
               <xref target="RFC9126">Pushed Authorization Requests</xref>
               provide an
-              interoperable way to push a Request Object
-              directly to the AS in exchange for a
+              interoperable way to push an authorization request
+              directly to the AS in exchange for a one-time-use
               <spanx style="verb">request_uri</spanx>.
+	      The standard PAR metadata parameters are used in the
+	      RP and OP metadata to indicate its use.
             </t>
+	    <t>
+	      When used with OpenID Federation,
+	      a Request Object MUST be sent to the PAR endpoint,
+	      signed with one of the RPs private keys.
+	      The corresponding public key MUST be in the RP's JWK Set.
+	      Furthermore, when used with OpenID Federation,
+	      the client authentication methods for the PAR endpoint
+	      are restricted to those proving possession of a private key.
+	      Likewise, the corresponding public key MUST be in the RP's JWK Set.
```

Comment:
It may be helpful to clarify that the keys referenced here are OIDC RP keys (i.e., published under metadata>openid_relying_party, and NOT Federation keys).